src/api/rest/providers/github/schemas/events.js

```diff
@@ -0,0 +1,32 @@
+import { schema } from 'normalizr';
+import { initSchema, toTimestamp } from 'utils';
+
+import { userSchema } from './users';
+import { orgSchema } from './orgs';
+import { repoSchema } from './repos';
+
+export const eventSchema = new schema.Entity(
+  'events',
+  {
+    actor: userSchema,
+    org: orgSchema,
+    repo: repoSchema,
+  },
+  {
+    idAttribute: event => event.id,
+    processStrategy: entity => {
```

Comment:
I think that can be refactor in order to avoid `processed.` repetition

```
proccessStrategy: entity => ({
    ...initSchema(),
   id: entity.id,
   ....
```

Reply:
Thanks a lot for your review man! Just went through your remarks and applied the suggestions. I didn't know about `String.includes`, nice one :)

I just kept EventSchema like that for the moment, cause it still needs some work that may require that `processed` const. All of this will be polished at the end of the PR.